### PR TITLE
fix: add shamilkotta

### LINKED ISSUE #1597: [PROFILE ADDITION] SHAMIL
### Profile screenshot

githubProf

### PR DESCRIPTION
**Pull request by shamilkotta**
resolves #1597

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/shamilkotta.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/64640025/209474929-8e286c45-3941-4990-977e-36bb78d55833.jpeg",
+  "issueId": 1597,
+  "name": "SHAMIL K",
+  "username": "shamilkotta"
+}
```
Comment (Panquesito7) [inline profiles/shamilkotta.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by shamilkotta** [comment]
changes updated please have a look 
Thank you

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**